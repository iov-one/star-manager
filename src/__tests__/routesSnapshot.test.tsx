import { shallow } from "enzyme";
import React from "react";
import Routes from "routes";
it("Renders the <Routes /> Component", () => {
  const component = shallow(<Routes />);
  expect(component).toMatchSnapshot();
});
