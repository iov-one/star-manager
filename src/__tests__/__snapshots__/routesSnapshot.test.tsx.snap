// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the <Routes /> Component 1`] = `<LoadingView />`;
